src/ShallowTraversal.js

```diff
@@ -18,8 +21,17 @@ import {
 export function childrenOfNode(node) {
   if (!node) return [];
   const maybeArray = propsOfNode(node).children;
+  const recursiveWithout = array =>
+    map(without(array, false, undefined, null), value => {
+      if (isArray(value)) {
+        return recursiveWithout(value);
+      }
+      return value;
+    });
+
+  const filtered = isArray(maybeArray) ? recursiveWithout(maybeArray) : maybeArray;
   const result = [];
-  React.Children.forEach(maybeArray, child => result.push(child));
+  React.Children.forEach(filtered, child => result.push(child));
```

Comment:
FYI, you can change the code and then use `git commit --amend` to combine the changes into the latest commit so that it stays as a single commit.